.github/workflows/ttnn-run-sweeps-new.yaml

```diff
@@ -0,0 +1,49 @@
+name: "ttnn - Run sweeps new"
+
+on:
+  # schedule:
+  #   - cron: "0 1,7,13,19 * * *"
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branch:
+      - jdesousa/sweeps-ci
```

Comment:
is this meant to target this branch on every push?

Reply:
Just temporary to test the workflow, will remove before merging.